srml/council/src/seats.rs

```diff
@@ -223,15 +307,22 @@ decl_module! {
 			#[compact] index: VoteIndex
 		) -> Result {
 			let who = ensure_signed(origin)?;
-			ensure!(!total.is_zero(), "stake deposited to present winner and be added to leaderboard should be non-zero");
+			ensure!(
+				!total.is_zero(),
+				"stake deposited to present winner and be added to leaderboard should be non-zero"
+			);
 
 			let candidate = T::Lookup::lookup(candidate)?;
 			ensure!(index == Self::vote_index(), "index not current");
 			let (_, _, expiring) = Self::next_finalize().ok_or("cannot present outside of presentation period")?;
-			let stakes = Self::snapshoted_stakes();
-			let voters = Self::voters();
-			let bad_presentation_punishment = Self::present_slash_per_voter() * BalanceOf::<T>::from(voters.len() as u32);
-			ensure!(T::Currency::can_slash(&who, bad_presentation_punishment), "presenter must have sufficient slashable funds");
+			// TODO: Most likely we prefer this bond to be proportional to `|voters| * |candidates|`.
+			let bad_presentation_punishment =
```

Comment:
@gavofyork regarding making this slash amount proportional to `|voters| * |candidates|`:

 I will assume that with chunked approvals, this is no longer valid: the number of candidates does not affect this call anymore. We always read one chunk that contains that particular candidate being presented. Based on the size of the chunk, the `Self::present_slash_per_voter()` can be increased if needed.